grafana/.gitignore

```diff
@@ -0,0 +1,4 @@
+/grafana-6.2.0.tar.gz
+/grafana_webpack-6.2.0.tar.gz
+/grafana-6.2.2.tar.gz
+/grafana_webpack-6.2.2.tar.gz
```

Comment:
This file should be in the rootdir?
Also, it's better to just put wildcards like grafana-*.tar.gz to avoid touch this file in updates?

Reply:
fixed.